Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM busybox
+
+# The commit sha we wish to ship to prod
+ENV SHA master
+
+RUN apk --update add curl
+
+RUN curl -fsSL https://github.com/mozilla-conduit/phabricator-extensions/archive/${SHA}.tar.gz -o extensions.tar.gz \
```

Comment:
This is a little strange that we're downloading the github archive of the project, instead of just adding it from the filesystem. You shouldn't actually need any RUN commands for this, just "ADD" or "COPY" directives that will put the code from the working directory inside.